### DISCUSSION TITLE
[Question] How to verify/wait/process for WebSocket response

### DISCUSSION THREAD

**Discussion by cuonghuunguyen** [artilleryio/artillery#1205] [Artillery]
I am trying to load test my **Meteor** server which is using DDP. 

Some problems I am encountering:
* Cannot wait for the previous request to be a success before continuing 
* Cannot get the  data from previous requests to use in the next request
* Cannot use an `auto-increase` field for each request (request-id)

Could you please help me to find the documentation which mentions how to do so?

**Comment by hassy** [comment]
hi @cuonghuunguyen, the WebSocket engine does not support waiting or request chaining

There's a Meteor-specific engine here: https://github.com/kschingiz/artillery-engine-meteor - which may be better suited for your tests (and easier to extend).
